src/contracts/core/RewardsCoordinator.sol

```diff
@@ -82,7 +84,9 @@ contract RewardsCoordinator is
         uint32 _MAX_FUTURE_LENGTH,
         uint32 _GENESIS_REWARDS_TIMESTAMP,
         uint32 _OPERATOR_SET_GENESIS_REWARDS_TIMESTAMP,
-        uint32 _OPERATOR_SET_MAX_RETROACTIVE_LENGTH
+        uint32 _OPERATOR_SET_MAX_RETROACTIVE_LENGTH,
+        uint32 _GENESIS_PERFORMANCE_REWARDS_TIMESTAMP,
```

Comment:
Expecting this to be the same, since performance based rewards are also operator set based. Don't make the change yet since still deciding.

Reply:
Okay wasn't clear on that, will fix.